checkout-example-advanced/views/layouts/main.handlebars

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+	<!-- Adyen JS from TEST environment (change to live for production)-->
+	<script src="https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/5.40.0/adyen.js"
+		 integrity="sha384-ds1t0hgFCe636DXFRL6ciadL2Wb4Yihh27R4JO7d9CF7sFY3NJE4aPCK0EpzaYXD"
+		 crossorigin="anonymous"></script>
+            
+	<!-- Adyen CSS from TEST environment (change to live for production)-->
+	<link rel="stylesheet"
+		 href="https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/5.40.0/adyen.css"
+		 integrity="sha384-BRZCzbS8n6hZVj8BESE6thGk0zSkUZfUWxL/vhocKu12k3NZ7xpNsIK39O2aWuni"
+		 crossorigin="anonymous">
+
+  <link rel="stylesheet" href="/css/application.css">
+
+  <title>Checkout Demo</title>
```

Comment:
Add: Checkout Demo `Advanced`

Reply:
I'm actually trying tokeep the templates in sync for future maintenance, but that's not necessarily a bad idea